tests/UnitTests/CoreGraphics.drawing/CGContextDrawing_PatternTests.cpp

```diff
@@ -370,4 +370,14 @@ DISABLED_DRAW_TEST_F(CGPatternTests, PatternDrawPath, UIKitMimicTest<>) {
     CGPathRelease(theFirstPath);
     CGPathRelease(theSecondPath);
 }
+
+DISABLED_DRAW_TEST_F(CGPatternTests, PatternFillWindowsLogoRotate, UIKitMimicTest<>) {
```

Comment:
doesn't work as expected. 
Bugs associated. 

---
In reply to: [101332405](https://github.com/Microsoft/WinObjC/pull/1952#discussion_r101332405) [](ancestors = 101332405)